### PR TITLE
Dockerfile for python2.7

### PR DESCRIPTION
**Pull request by spesnova**
## WHAT

Docker Image for Python 2.7

You can test this

``` bash
$ script/test
```
## REF
- https://github.com/docker-library/python

### CODE REVIEW

**Comment by spesnova** [comment]
@dtan4 plz review!

File: script/test [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+BASE_DIRECTORY=`pwd`
+
+echo "==> Building target..."
+cd 2.7
+docker build -t target .
+cd "${BASE_DIRECTORY}"
+
+echo "==> Running test..."
+cd 2.7
+docker run --rm -it python python -V | grep "Python 2.7.9" > /dev/null 2>&1
```
Comment (dtan4) [inline script/test]: This command uses Docker Hub's Python image...
Is this a mistake of `docker run --rm -it target python -V` ?

Comment (spesnova) [inline script/test]: fixed :+1:

**Comment by dtan4** [comment]
LGTM :+1:

**Pull request merged**